runtime/common/KernelWrapper.h

```diff
@@ -0,0 +1,347 @@
+/****************************************************************-*- C++ -*-****
+ * Copyright (c) 2022 - 2023 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+#pragma once
+#include "cudaq/platform.h"
+#include "cudaq/utils/registry.h"
+#include <cstdint>
+#include <cstring>
+#include <string>
+#include <vector>
+
+/*! \file KernelWrapper.h
+    \brief Utility classes to support library-mode kernel launch.
+
+    This header file defines classes supporting launching quantum kernels in
+   library mode (without the bridge) when using the remote platform.
+    Specifically, it provides a layer of indirection, `cudaq::invokeKernel`,
```

Comment:
Doesn't using a blank line to separate the paragraphs work here? (The extra indentation might be seen as a "bug" or autocorrected by code formatting tools with bugs.)